solus_sc/updates_view.py

```diff
@@ -276,6 +277,53 @@ def load_updates(self):
         self.basket.invalidate_all()
         GObject.idle_add(self.init_view)
 
+    def draw_rounded(self, cr, x, y, width, height, radius):
+        corner_radius = height / 10.0;
+        degrees = pi / 180.0;
+
+        cr.arc(x + width - radius, y + radius, radius, -90 * degrees, 0 * degrees);
+        cr.arc(x + width - radius, y + height - radius, radius, 0 * degrees, 90 * degrees);
+        cr.arc(x + radius, y + height - radius, radius, 90 * degrees, 180 * degrees);
+        cr.arc(x + radius, y + radius, radius, 180 * degrees, 270 * degrees);
+
+    def create_icon_with_number(self, icon_in, count):
+        surface = cairo.ImageSurface (cairo.FORMAT_ARGB32, icon_in.get_width(), icon_in.get_height())
+        context = cairo.Context(surface)
+
+        Gdk.cairo_set_source_pixbuf(context, icon_in, 0, 0)
+        context.paint()
+
+        context.select_font_face("Clear Sans", cairo.FONT_SLANT_NORMAL, cairo.FONT_WEIGHT_BOLD)
```

Comment:
Yeah I made that initial note about hard coded values and not sure where to set/store them. For the font I will just use the system font.